examples/tutorial_credit_scoring.ipynb

```diff
@@ -55,7 +62,17 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Install AIF360\n",
+    "!pip install 'aif360[lime]'"
```

Comment:
this should just be `!pip install 'aif360'`

Reply:
you'll also need to download the german credit score dataset, I believe. See the instructions here:
https://github.com/Trusted-AI/AIF360/tree/master/aif360/data/raw/german